python/bindings/include/openravepy/openravepy_kinbody.h

```diff
@@ -152,6 +152,7 @@ typedef OPENRAVE_SHARED_PTR<PyGrabbedInfo> PyGrabbedInfoPtr;
     py::object _name = py::none_();
 #endif
     py::object _transform = ReturnTransform(Transform());
+    bool _isRobot = false;
```

Comment:
@ziyan is it necessary to update MINOR version just because `_isRobot` was added here?

Reply:
Incremented PATCH version instead.